beta-src/src/components/map/components/WDArrowContainer.tsx

```diff
@@ -394,15 +430,15 @@ function accumulateStandoffMarks(
     if (!rootTerritory) return;
 
     const [x1, y1, w1, h1] = getTargetXYWH("territory", rootTerritory);
-    const MARKSIZE = 25;
+    const MARKSIZE = 15;
     arrows.push(
       <line
         key={`standoffmark-${province}-1`}
         x1={x1 - MARKSIZE}
         y1={y1 - MARKSIZE}
         x2={x1 + MARKSIZE}
         y2={y1 + MARKSIZE}
-        stroke="red"
+        stroke="tomato"
```

Comment:
Is this some kind of joke?

Reply:
Hehe. I looked up a list of standard svg color names, there's like a fixed set of them. The shade of reddish-orange I was looking for happened to be named "tomato".